### ISSUE TITLE
Permission to read any sites data, not just YouTube

### ISSUE THREAD

**Issue by griknok** [animeshkundu/youtube-audio#33]
Other extensions like Enhancer for YouTube only ask for access to youtube.com but this extension can read all site's data! Why?

**Comment by Sytten** [comment]
Yeah it made me pause before installing it

**Comment by MemeticsX** [comment]
Agreed; please revise the permissions to limit the app's scope to where it belongs.  I want to install this but won't until the permissions are set up appropriately.
